src/components/HTMLEngineProvider/HTMLRenderers/MentionReportRenderer.tsx

```diff
@@ -61,7 +59,7 @@ function MentionReportRenderer({style, tnode, TDefaultRenderer, reports, ...defa
     const htmlAttributeReportID = tnode.attributes.reportid;
 
     const currentReportID = useCurrentReportID();
-    const currentReport = getReport(currentReportID?.currentReportID);
+    const [currentReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${currentReportID?.currentReportID}`);
```

Comment:
`getReport` returns either the report or the draft report. Are we sure that in this case we are only interested in the report? (Please double check on other instances too). You may refer to https://github.com/Expensify/App/issues/40668 and its PRs to see where we may need to fallback to the draft report.

Reply:
@s77rt I checked the mentioned PR when investigating this PR. The draft report is only used when we move transaction from track expense. It will be cleared in optimistic data when we confirm to move the transaction from track expense and it's not used to display any other UI of the App.